### PR TITLE
Fixed alert test

### PR DESCRIPTION
**Pull request by nidhi-soni1104**
Fixed Failing test case

### CODE REVIEW

**Review by akostadinov — APPROVED** [review]
lgtm, hopefully @hallelujah can confirm this is what he meant as this is test code, not application code. Returned values are sometimes good to test. But checking return type of a model scope is over the top IMHO.

**Review by hallelujah — CHANGES_REQUESTED** [review]
Yes ... well you can even test that the only record is the one you just created instead of testing the count. Which will make it more clearer IMO ;)

**Comment by nidhi-soni1104** [comment]
> Yes ... well you can even test that the only record is the one you just created instead of testing the count. Which will make it more clearer IMO ;)

I think the count was also clear to me, but I updated the test and checked the actual created object.

File: test/unit/alert_test.rb [hunk 1]
```diff
@@ -15,9 +15,8 @@ def test_by_level
     FactoryBot.create(:limit_alert, level: 50)
     FactoryBot.create(:limit_alert, level: 80)
 
-    alerts = Alert.by_level(80)
-    assert_kind_of Alert::ActiveRecord_Relation, alerts
-    assert_equal 1, alerts.count
+    alert = Alert.by_level(80).last
```
Comment (hallelujah) [inline test/unit/alert_test.rb:18]: What about

```
assert_equal [alert], Alert.by_level(80).to_a
```

Comment (nidhi-soni1104) [reply test/unit/alert_test.rb:18]: Done

Comment (hallelujah) [reply test/unit/alert_test.rb:18]: well not really what I expected :) you need to modify the line 16 into

```
alert = FactoryBot.create(:limit_alert, level: 80)
```

**Review by hallelujah — CHANGES_REQUESTED** [review]

**Comment by nidhi-soni1104** [comment]
@hallelujah test meaning was same but its a Good suggestion, Changes are done.

**Review by hallelujah — APPROVED** [review]

File: test/unit/alert_test.rb [hunk 2]
```diff
@@ -12,12 +12,8 @@ class AlertTest < ActiveSupport::TestCase
   should validate_numericality_of(:utilization)
 
   def test_by_level
-    FactoryBot.create(:limit_alert, level: 50)
-    FactoryBot.create(:limit_alert, level: 80)
-
-    alerts = Alert.by_level(80)
-    assert_kind_of Alert::ActiveRecord_Relation, alerts
-    assert_equal 1, alerts.count
+    alert = FactoryBot.create(:limit_alert, level: 80)
```
Comment (hallelujah) [inline test/unit/alert_test.rb:15]: Wait, why did you remove the level 50 one? Does not that change the meaning of the test?

**Review by hallelujah — CHANGES_REQUESTED** [review]

File: test/unit/alert_test.rb (hunk 2 above)
Comment (nidhi-soni1104) [reply test/unit/alert_test.rb:15]: I think No, because we are not even using it anywhere we have just created it, It was useful when we are checking count.

Comment (nidhi-soni1104) [reply test/unit/alert_test.rb:16]: Cleared doubt with @akostadinov and sorted the PR. @hallelujah

**Review by akostadinov — APPROVED** [review]

**Review by hallelujah — APPROVED** [review]
